dice_ml/diverse_counterfactuals.py

```diff
@@ -114,7 +114,7 @@ def display_df(self, df, show_only_changes):
                         newdf[ix][jx] = '-'
                     else:
                         newdf[ix][jx] = str(newdf[ix][jx])
-            display(pd.DataFrame(newdf, columns=df.columns))  # works only in Jupyter notebook
```

Comment:
How does the notebook output look like with this change?

Reply:
The index gets printed in the jupyter notebook